internal/output/output.go

```diff
@@ -73,6 +73,13 @@ func (v VerificationStatus) addToSuccesses(successes []output.Result) []output.R
 	return result
 }
 
+type Attestation struct {
```

Comment:
Could we use the existing type `internal/attestation.Attestation` here?

Reply:
No, for a couple of reasons. 

First, `output.Attestation` is a struct that **represents** an attestation. It is not the attestation itself. The `output` package seems like the right place for this. This could've been called `AttestationOutput` but given that it resides in the package `output`, that seemed redundant. 

Second, `attestation.Attestation` is an interface. It is not possible to add attributes to an interface in go afaik. Instead, I added a method to the interface to produce the struct that represents the attestation. 

I don't really see a problem with this approach. The separation of concerns is clearly defined to me, but I'm always open to suggestions.